Lib/test/test_os.py

```diff
@@ -3524,6 +3525,48 @@ def test_memfd_create(self):
         self.assertFalse(os.get_inheritable(fd2))
 
 
+@unittest.skipUnless(hasattr(os, 'eventfd'), 'requires os.eventfd')
+@support.requires_linux_version(2, 30)
```

Comment:
I think the test class should be ran with Linux 2.27+ (to at least cover `test_eventfd_initval`) and then `test_eventfd_semaphore` (or any future tests that require use of the `os.EFD_NONBLOCK` flag) can require 2.30+. I'm not sure that it's entirely necessary or the number of devices it would actually be applicable to, but it seems like a better practice to have *some* amount of test coverage for 2.27 - 2.29.

Also, would it be worthwhile to add a test that makes use of select (maybe for poll and epoll as well, via mixin), which uses them to check if the file descriptor is ready for reading or writing? I suspect that will be a primary use case `os.eventfd()`, so ensuring that behavior works as expected would make sense to me. I'm just not certain if it makes sense to include those in `test_os`, or if there might be a better location.

Reply:
It's actually 2.6.30, not 2.30. I'll fix that with the next commit. Linux Kernel 2.6.30 was released in 2009. It's highly unlikely that somebody is running a Kernel version between 2.6.27 and 2.6.30.

Reply:
> It's actually 2.6.30, not 2.30. I'll fix that with the next commit.

Ah, good catch. I completely didn't notice the missing `.6`, despite recently reading over the man page for `eventfd()`. :-)

> It's highly unlikely that somebody is running a Kernel version between 2.6.27 and 2.6.30.

Yeah, after looking over the dates for the releases, I'm now in agreement that it would be highly unlikely. If anything, I suspect most on the 2.6.x branch would be using 2.6.39 or 2.6.32.

(Based on https://upload.wikimedia.org/wikipedia/en/timeline/40658da696ee210d2be5f221ac12e43b.png.)